v2/lib/webpack/base.js

```diff
@@ -51,7 +51,7 @@ module.exports = function createBaseConfig(props, isServer) {
       .loader('babel-loader')
       .options({
         babelrc: false,
-        presets: ['env', 'react'],
+        presets: ['@babel/env', '@babel/react'],
         plugins: [isServer ? 'dynamic-import-node' : 'syntax-dynamic-import'],
```

Comment:
Haven't really tried this out but i think the dynamic import node babel plugin is not compatible with babel 7.

See https://github.com/airbnb/babel-plugin-dynamic-import-node/pull/64

Bappening in CRA as well

https://github.com/facebook/create-react-app/pull/4984

Reply:
@fiennyangeln Can you verify this? What do we do about this then?

Reply:
@yangshun yes, the PR for babel 7 upgrade for that plugin is still open. How about using https://www.npmjs.com/package/babel-plugin-transform-dynamic-import instead?